haas/api.py

```diff
@@ -343,9 +344,8 @@ def node_detach_network(node, nic, network):
     db.session.commit()
     return '', 202
 
-
-                            # Head Node Code #
-                            ##################
+    # Head Node Code #
+    ##################
```

Comment:
Didn't realize most of these were indented one. They were meant to be top level headings, so they should be all the way to the left. The tools are okay with this because they think it's inside node_detach_network.